django/views/templates/default_urlconf.html

```diff
@@ -65,213 +50,113 @@
-            }
-          }
-          @keyframes smokeFirefox {
-            0% {
-              -webkit-transform: scale(1.58, 1.2) translateY(-75px) skew(0);
-              transform: scale(1.58, 1.2) translateY(-75px) skew(0);
+              transform: translate3d(-5px, 0, 0);
             }
             100% {
-              -webkit-transform: scale(1.58, 1.21) translateY(-75px) skew(1deg);
-              transform: scale(1.58, 1.21) translateY(-75px) skew(1deg);
+              transform: translate3d(5px, 0, 0);
             }
           }
-          .flame 0%, .flame 100% {
-            -webkit-transform: translate(265px, 249px) scale(1.9) rotate(180deg);
-            transform: translate(265px, 249px) scale(1.9) rotate(180deg);
-          }
           .flame {
-            -webkit-animation: burnInner2 .1s infinite ease-in-out alternate;
-            animation: burnInner2 .1s infinite ease-in-out alternate;
-          }
-          @-webkit-keyframes burnInner2 {
-            0% {
-              -webkit-transform: translate(265px, 249px) scale(1.9) rotate(180deg) skew(5deg);
-              transform: translate(265px, 249px) scale(1.9) rotate(180deg) skew(5deg);
-            }
-            100% {
-              -webkit-transform: translate(265px, 253px) scale(2.2) rotate(180deg) skew(-5deg);
-              transform: translate(265px, 253px) scale(2.2) rotate(180deg) skew(-5deg);
-            }
+            animation: burnInner2 .1s 70 ease-in-out alternate;
           }
           @keyframes burnInner2 {
             0% {
-              -webkit-transform: translate(265px, 249px) scale(1.9) rotate(180deg) skew(5deg);
-              transform: translate(265px, 249px) scale(1.9) rotate(180deg) skew(5deg);
+              transform: translate3d(0, 0, 0);
             }
             100% {
-              -webkit-transform: translate(265px, 253px) scale(2.2) rotate(180deg) skew(-5deg);
-              transform: translate(265px, 253px) scale(2.2) rotate(180deg) skew(-5deg);
+              transform: translate3d(0, 3px, 0);
             }
           }
-          @-webkit-keyframes thurst {
+          @keyframes thurst {
             0% {
               opacity: 1;
             }
             100% {
               opacity: .5;
             }
           }
-          @keyframes thurst {
-            0% {
-              opacity: 1;
-            }
-            100% {
-              opacity: .5;
+          @media (prefers-reduced-motion: reduce) {
+            .exhaust__line,
+            .smoke,
+            .flame {
+              animation: none;
             }
           }
-          h2 {
-            font-size: 22px;
-            max-width: 500px;
+          h1 {
+            font-size: 1.375rem;
+            max-width: 32rem;
             margin: 5px auto 0;
           }
           main p {
-            font-size: 16px;
-            line-height: 20px;
-            max-width: 390px;
+            line-height: 1.25;
+            max-width: 25rem;
```

Comment:
It felt sensible to redefine those max-widths based on the font size, so they keep up with any zoom of text size.